kie-server-parent/kie-server-tests/kie-server-integ-tests-all/src/test/java/org/kie/server/integrationtests/jbpm/FormServiceIntegrationTest.java

```diff
@@ -124,6 +129,29 @@ protected void runGetTaskFormTest( long processInstanceId ) {
         }
     }
 
+    @Test(expected = KieServicesException.class)
```

Comment:
`expected = KieServicesException.class` should be removed now when the exception is processed in the test.